volplugin/handlers.go

```diff
@@ -226,6 +227,7 @@ func mount(master, host string, ttl int) func(http.ResponseWriter, *http.Request
 
 		actualSize, err := volConfig.Options.ActualSize()
 		if err != nil {
+			removeStopChan(uc.Request.Name)
```

Comment:
If we do that it gets cleaned up here too: https://github.com/erikh/volplugin/blob/global-configuration/volplugin/handlers.go#L261

the stop chan (and the heartbeat)n is held open while the volumes are mounted, so we can't do that.